alacritty/src/event.rs

```diff
@@ -1964,19 +2022,32 @@ impl input::Processor<EventProxy, ActionContext<'_, Notifier, EventProxy>> {
                         self.key_input(event);
                     },
                     WindowEvent::ModifiersChanged(modifiers) => self.modifiers_input(modifiers),
-                    WindowEvent::MouseInput { state, button, .. } => {
-                        self.ctx.window().set_mouse_visible(true);
-                        self.mouse_input(state, button);
+                    WindowEvent::PointerButton { state, position, button, .. } => {
+                        if let ButtonSource::Touch { finger_id, .. } = button {
```

Comment:
Why is a touch event a pointer button? That seems like an awful API.

Reply:
It's to follow web naming, and move everything into the same input. In general, pointer something that points, so touch is a pointer.

Reply:
We also have all pointer-ish events in `Pointer` so apps that don't care about touch/tablet actually support touch/tablet now oob. We also support tablet input now with this.